samtranslator/policy_templates_data/policy_templates.json

```diff
@@ -2360,6 +2360,34 @@
           }
         ]
       }
+    },
+    "StepFunctionsSyncExecutionPolicy": {
```

Comment:
Can we have a better name that mentions it is for Synchronous Express Workflows? It is also using the same name as in PR #1622 which covers the [standard synchronous workflow](https://docs.aws.amazon.com/step-functions/latest/dg/stepfunctions-iam.html#synchronous).